plugins/notebook/base.tid

```diff
@@ -217,7 +217,7 @@ h1.tc-site-title {
   text-overflow: ellipsis;
   padding-top: 10px;
   z-index: 500;
-  white-space: nowrap;
+  white-space: normal;
```

Comment:
The problem is that most wrapping elements in the sidebar will look bad on multiple lines, like the screenshot below:

![Screenshot from 2020-12-20 16-25-19](https://user-images.githubusercontent.com/123539/102717058-26715080-42e0-11eb-95ba-ad8fddaafae3.png)

I'll see if enabling white-space wrapping works with some tweaks.

Reply:
Instead of setting it to `normal`, this line can be removed as it's the default value.